README.md

```diff
@@ -1,9 +1,10 @@
-### debugger.html
+### Debugger.html
 
 This is a prototype debugger written without any XUL and based on React and Redux.
 
 [![Build Status](https://travis-ci.org/jlongster/debugger.html.svg?branch=master)](https://travis-ci.org/jlongster/debugger.html)
 
+----
```

Comment:
It was feeling long. I'm not a big fan of them. I'll delete them, the better solution is probably to break some of the content out into other doc files